File: Service/FinCare/RMA.Service.FinCare.Contracts/Entities/Commissions/CommissionClawBackAccount.cs
using RMA.Common.Entities;

using System.Collections.Generic;

namespace RMA.Service.FinCare.Contracts.Entities.Commissions
{
    public class CommissionClawBackAccount : AuditDetails
    {
        public int ClawBackAccountId { get; set; }
        public int RecepientTypeId { get; set; }
        public int RecepientId { get; set; }
        public string RecepientCode { get; set; }
        public string RecepientName { get; set; }
        public decimal AccountBalance { get; set; }
        public List<CommissionClawBackAccountMovement> ClawBackAccountMovements { get; set; }
    }
}

File: Web/RMA-Web-Shell/projects/fincare/src/app/payment-manager/models/commission-clawback-account.ts
import { BaseClass } from 'projects/shared-models-lib/src/lib/common/base-class';
import { CommissionClawBackAccountMovement } from './commission-clawback-account-movement';

export class CommissionClawBackAccount extends BaseClass {
    ClawBackAccountId: number;
    recepientTypeId: number;
    recepientId: number;
    recepientCode: string;
    recepientName: string;
    accountBalance: number;
    headerStatusId: number;
    clawBackAccountMovements: CommissionClawBackAccountMovement[];
}
